storage/src/storage.rs

```diff
@@ -6,13 +6,14 @@ use jmt::{
     storage::{Node, NodeBatch, NodeKey, TreeReader, TreeWriter},
     WriteOverlay,
 };
+use metrics::gauge;
 use rocksdb::{Options, DB};
 use tokio::sync::RwLock;
 use tracing::{instrument, Span};
 
 use penumbra_tct as tct;
 
-use crate::State;
+use crate::{metrics::TCT_SIZE_BYTES, State};
```

Comment:
Rather than importing everything from two places, if we just do `use crate::metrics;`, we can write code like
```rust
metrics::gauge!(metrics::TCT_SIZE_BYTES, /* ... */);
```
and then it's clear that all of the constants are metrics-related, rather than being part of some other code.